Entity/SelectedStopPointRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace CanalTP\MttBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+/**
+ * SelectedStopPointRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class SelectedStopPointRepository extends EntityRepository
```

Comment:
I see lot of empty repositories like : 
* AreaPdfRepository
* AreaRepository
* FrequencyRepository
* LayoutCustomerRepository
* LineTimetableRepository
* TemplateRepository

Should I delete all of them then ?